invenio_search_ui/ext.py

```diff
@@ -49,6 +49,11 @@ def init_app(self, app):
     def init_config(self, app):
         """Initialize configuration."""
         app.config.setdefault(
-            "SEARCH_UI_BASE_TEMPLATE",
-            app.config.get("BASE_TEMPLATE",
-                           "invenio_search_ui/base.html"))
+            'SEARCH_UI_BASE_TEMPLATE', 'invenio_theme/page.html'
```

Comment:
IMHO If we do it this way then we need to dublicate settings for `requirejs` (ie `build.js` and `settings.js`)